### ISSUE TITLE
learning rate 

### ISSUE THREAD

**Issue by csm-kr** [mozhuangb/SiameseFC-pytorch#5]
Hi, thank you for your code!

we have some questions about this repo! :)

This repo's train learning rate is set to 1e-4, but the original paper says that 1e-2 to 1e-5

So we experiment with the 1e-2 lr but, the loss don't converge and has nan loss value.

So why do you set the learning rate at 1e-4?
